compiled_starters/ruby/app/server.rb

```diff
@@ -0,0 +1,24 @@
+require 'socket'
+
+def start_server
+  udp_socket = UDPSocket.new
+  udp_socket.bind('127.0.0.1', 2053)
+
+  # You can use print statements as follows for debugging, they'll be visible when running tests.
+  print("Logs from your program will appear here!")
+
+  # Uncomment this block to pass the first stage
+  #
+  # loop do
+  #   begin
+  #     buf, source = udp_socket.recvfrom(512)
+  #     response = ''
+  #     udp_socket.send(response, 0, source[3], source[1])
```

Comment:
Can we use better variable names here? Isn't clear what `source[3]` and `source[1]` are.

Reply:
Not sure if pattern matching in Ruby works this way, but maybe we could do something like: 

> message, [_, sender_port, _, sender_address] = socket.recvfrom(1024)